plugins/outputs/elasticsearch/elasticsearch.go

```diff
@@ -475,6 +495,32 @@ func (a *Elasticsearch) GetIndexName(indexName string, eventTime time.Time, tagK
 	return fmt.Sprintf(indexName, tagValues...)
 }
 
+func (a *Elasticsearch) GetPipelineName(pipelineInput string, tagKeys []string, metricTags map[string]string) string {
```

Comment:
I think we do not need to export this function
```suggestion
func (a *Elasticsearch) getPipelineName(pipelineInput string, tagKeys []string, metricTags map[string]string) string {
```